awx/ui/src/screens/Job/JobDetail/JobDetail.js

```diff
@@ -428,6 +427,7 @@ function JobDetail({ job, inventorySourceLabels }) {
                 ))}
               </ChipGroup>
             }
+            isEmpty={credentials.length === 0}
```

Comment:
you should be able to use something like `isEmpty={!credentials || credentials.length === 0}` and get rid of the `credentials &&` check on line 408

Reply:
...and similar for some of the other Details in this file